src/v0/sources/adjust/utils.js

```diff
@@ -0,0 +1,14 @@
+const { TransformationError } = require('@rudderstack/integrations-lib');
+
+const processTimestamp = (rawTimestamp) => {
```

Comment:
Better name would be `getTimestamp` I believe. Let's make this minor change

Reply:
should we keep in the outside utils? can this be useful in other destination built down the line?

Reply:
Let's keep it in `adjust` itself. In future when there is a requirement, let's move it to the utils outside